modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -204,8 +149,7 @@ include::{examplesdir}installation/lets_encrypt/list.sh[]
 This script:
 
 * Renews all your issued certificates.
-* In case you have enabled the post hook for your webserver in `cli.ini`, it will reload
-the web server configuration automatically if a certificate has been renewed.
+* In case you have enabled the post hook for your webserver in `cli.ini`, it will reload the web server configuration automatically if a certificate has been renewed.
```

Comment:
```suggestion
* In case you have enabled the post hook for your web server in `cli.ini`, it will reload the web server configuration automatically if a certificate has been renewed.
```